corehq/util/timezones/conversions.py

```diff
@@ -0,0 +1,113 @@
+import pytz
+from corehq.const import USER_DATETIME_FORMAT
+from corehq.util.dates import safe_strftime
+
+TIMEZONE_DATA_MIGRATION_COMPLETE = False
```

Comment:
A brief comment here would be good just outlining what's expected to change. Not essential though.